### ISSUE TITLE
Examples not working 

### ISSUE THREAD

**Issue by abhi94** [detrout/qt-gstreamer#2]
gstreamer-1.4.5 
gst-plugins-base-1.4.5
installed from tarballs 
then,  installed qt-gstreamer-1.2.0 :
$ mkdir build && cd build
$ cmake .. -DQTGSTREAMER_EXAMPLES=[ON]  -DCMAKE_INSTALL_PREFIX=/path/to/installation/prefix
$ make
$ make install

the examples are built successfully, but there is no video playback when runnig

**Comment by detrout** [comment]
Since its been a while. I wanted to say I saw your report I just haven't had a chance to investigate.

**Comment by detrout** [comment]
I had a chance to look. The examples run in the build tree. Do they work for you in the build tree?

**Comment by vdehors** [comment]
As examples for player (player & qmlplayer) uses playbin element, you need all elements required to decode your video, which can be provided by others packages than gst-plugins-base-1.4.5 (depending on your video). Having the element "playbin" may not be enough to decode it.

Does the playback works using gst-launch ?

**Comment by abhi94** [comment]
@detrout : Nope, It didn't work in the build tree. I might missed some dependency??  Thank you for your help. I will try again.
@vdehors : I didn't aware of gst-launch. what should I do to make this work with gst-launch ? I assume apt-get install gstreamer-tools will provide gst-launch. Or should I compile it from source ?
Thank you very much.

**Comment by vdehors** [comment]
I think gst-launch is provided by your tarball of gstreamer1.0. To check the playback of your video, you can launch :
> gst-launch-1.0 playbin uri="file://[path to your video]"

Note that if you give an absolute path, there is three '/' after "file:". Ex: file:///home/user/video.avi.

If the playback failed because of a gstreamer element missing, you should have an error. If the playback works, it may be a issue related to qtgstreamer.

**Comment by detrout** [comment]
Also  abhi94, what OS are your running? Did you try both qmlplayer and player? Did you build against Qt4 or Qt5?
